docs/source/docs/user-manual/command-reference/checkout.md

```diff
@@ -1,8 +1,4 @@
----
-title: 'Checkout'
-linkTitle: 'checkout'
-description: ''
----
+# checkout
```

Comment:
Please capitalize the title. There are also so many below.

Reply:
It shows the original command. If I change it to capital, then menus will show as capital.